UndertaleModTool/MainWindow.xaml.cs

```diff
@@ -1155,11 +1155,17 @@ public string GenerateMD5(string filename)
         private async Task LoadGMLCache(string filename, LoaderDialog dialog = null)
         {
             await Task.Run(() => {
-                if (SettingsWindow.UseGMLCache && File.Exists(Path.Join("GMLCache", "index")))
+                if (SettingsWindow.UseGMLCache)
                 {
+                    string cacheDirPath = Path.Join(ExePath, "GMLCache");
+                    string cacheIndexPath = Path.Join(cacheDirPath, "index");
```

Comment:
Why Path.Join instead of Path.Combine here?

Reply:
IIRC, because I thought that `Join()` will be enough here, as `Combine()` does more work.

Reply:
Combine does more work, but it's also a little safer, and considering that the rest of the codebase uses combine consistency would be nice here.

Reply:
Yeah, so I'll change this.